src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -86,26 +88,35 @@ const TopicPage = ({ pageData }) => {
             {description && <TopicDescription>{description}</TopicDescription>}
           </div>
           {curations.map(
-            ({
-              visualProminence,
-              summaries,
-              curationId,
-              title: curationTitle,
-              link,
-              position,
-            }) => (
-              <Curation
-                headingLevel={curationTitle && 3}
-                key={curationId}
-                visualStyle={VISUAL_STYLE.NONE}
-                visualProminance={visualProminence}
-                promos={summaries}
-                title={curationTitle}
-                topStoriesTitle={topStoriesTitle}
-                position={position}
-                link={link}
-                curationLength={curations && curations.length}
-              />
+            (
+              {
+                visualProminence,
+                summaries,
+                curationId,
+                title: curationTitle,
+                link,
+                position,
+                visualStyle,
+              },
+              index,
+            ) => (
+              // eslint-disable-next-line react/no-array-index-key
+              <React.Fragment key={`${curationId} ${index}`}>
+                {visualStyle === VISUAL_STYLE.BANNER && !isLive() && (
```

Comment:
Let's extract this into a function - we'll also need to add a test scenario if the visual style is banner but prominence is high - this should not render the message banner
```suggestion
                {shouldRenderMessageBanner(visualStyle, visualProminence) ? (
```

Reply:
Refactored but we will need to add a test - might need to rewrite the existing ones to include visualProminence in the base case along with visualStyle